DuckDuckGo/SiteRatingView.swift

```diff
@@ -38,27 +38,19 @@ public class SiteRatingView: UIView {
         let view = Bundle.main.loadNibNamed("SiteRatingView", owner: self, options: nil)![0] as! UIView
         self.addSubview(view)
         view.frame = self.bounds
-        addUserDefaultsObserver()
-    }
-    
-    deinit {
-        removeUserDefaultsObserver()
+        addContentBlockerConfigurationObserver()
```

Comment:
Does the `SiteRatingView` need to observe this? If the page is reloaded won't it update anyway via `webpageDidFinishLoading` in the view controller?

Reply:
The reload can take a long time so we want to update it immediately. We don't the user staring at an exclamation mark which suggests that the content blocker is off when they have switched it back on.